Source/Game/Network/Packets/InspectPackets.cs

```diff
@@ -270,6 +270,7 @@ public void Write(WorldPacket data)
             data.WriteUInt8(Face);
             data.WriteUInt8(Race);
             data.WriteUInt8(ClassID);
+            data.FlushBits();
```

Comment:
That flush should not be required. There is no bit writing happening here and these write functions also call a flush